src/test/scala/WebhookOpSpec.scala

```diff
@@ -0,0 +1,112 @@
+
+import org.scalatest.FunSpec
+import org.scalatest.BeforeAndAfter
+import scala.concurrent.Await
+
+import codecheck.github.models.Webhook
+import codecheck.github.models.WebhookConfig
+import codecheck.github.models.WebhookInput
+import codecheck.github.models.WebhookUpdateInput
+
+
+class WebhookOpSpec extends FunSpec with Constants with BeforeAndAfter {
+
+	val gURL = generateRandomWord;
```

Comment:
It is better to use valid github webhook application.
You can use 

http://github-hook.herokuapp.com/hook